backend/alembic/versions/2024_09_24_0718-cfc217bbdd23_add_status_column_into_chat_table.py

```diff
@@ -0,0 +1,36 @@
+"""Add status column into chat table
+
+Revision ID: cfc217bbdd23
+Revises: 33e313a32ce6
+Create Date: 2024-09-24 07:18:13.207997
+
+"""
+
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+import sqlmodel  # noqa
+
+
+# revision identifiers, used by Alembic.
+revision: str = "cfc217bbdd23"
+down_revision: Union[str, None] = "33e313a32ce6"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
```

Comment:
I don't think we should add a status column. There is no good way to keep track of sending status when web clients can come and go. That also removes the need to update the status in callbacks on the socket server.